packages/components/src/Link/_docs/Link.doc.stories.tsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react'
+import { type Meta, type StoryObj } from '@storybook/react'
+import { ExternalLinkIcon, FlagOffIcon } from '~components/Icon'
```

Comment:
We need to replace with these usages with the RC icon Icon component - this is why the external link icons look larger than they should be